code/model/modelread.cpp

```diff
@@ -4607,12 +4607,12 @@ void model_set_instance(int model_num, int sub_model_num, submodel_instance_info
 	if (flags & SSF_NO_DISAPPEAR) {
 		// If the submodel is to not disappear when the subsystem is destroyed, we simply
 		// make the submodel act as its own replacement as well
```

Comment:
This comment is now incorrect and should be fixed to explain the new behavior instead of the old.